broker/src/main/java/org/apache/rocketmq/broker/processor/ReplyMessageProcessor.java

```diff
@@ -173,6 +173,7 @@ private PushReplyResult pushReplyMessage(final ChannelHandlerContext ctx,
         replyMessageRequestHeader.setProperties(requestHeader.getProperties());
         replyMessageRequestHeader.setReconsumeTimes(requestHeader.getReconsumeTimes());
         replyMessageRequestHeader.setUnitMode(requestHeader.isUnitMode());
+        replyMessageRequestHeader.setBname(requestHeader.getBname());
```

Comment:
`bname` is arcane and inconsistent with the existing field naming convention.  Why not use brokerName?

Reply:
+1

Reply:
It's backport from 5.0 branch. The `bname` field is introduced by https://github.com/apache/rocketmq/commit/b6ff649291453d3dc2d702fbba1548d991b610ac.